File: app/src/at-tamle/java/asiantech/internship/winter/ex15/twice.java
package asiantech.internship.winter.ex15;

import java.util.Scanner;
//   liet ke cac ptu trong day xuat hien 2 lan
public class twice {
    public static int process(){
        Scanner input= new Scanner(System.in);
        boolean check= false;
        int n=0;
        while(!check){
            System.out.print(" ");
            try{
                n= input.nextInt();
                check= true;
            }catch(Exception e){
                System.out.println("Ban phai nhap so! hay nhap lai...");
                input.nextLine();
            }
        }
        return (n);
    }
    private static int countElement(int a[], int n, int i){
        int count= 0;
        for(int j=0 ; j<n ; j++){
            if(a[j]== i)
                count ++;
        }
        return (count);
    }
    public static void main(String[] args) {
        int n,i;
        System.out.println("Nhap n= ");
        n= process();
        int[] array= new int[n];
        for(i=0 ; i<n ; i++){
            System.out.print("Nhap phan tu thu " +(i+1)+" ");
            array[i]= process();
        }
        System.out.println("Cac phan tu trong day xuat hien 2 lan: ");
        for(i=0 ; i<n ; i++){
            if(countElement(array, n, array[i])==2 && countElement(array, i, array[i])==0){
                System.out.print(" "+array[i]);
            }
        }

    }
}

File: app/src/at-tamle/java/asiantech/internship/summer/kotlin/Twice.kt
package asiantech.internship.summer.kotlin

import java.util.Scanner

object Twice {
    private fun process(): Int {
        val input = Scanner(System.`in`)
        var check = false
        var n = 0
        while (!check) {
            try {
                n = input.nextInt()
                check = true
            } catch (e: Exception) {
                println("hay nhap lai")
                input.nextLine()
            }
        }
        return n
    }

    private fun countElement(a: IntArray, n: Int, i: Int): Int {
        var count = 0
        for (j in 0 until n) {
            if (a[j] == i)
                count++
        }
        return count
    }

    @JvmStatic
    fun main(args: Array<String>) {
        val n: Int
        var i: Int
        println("Nhap n= ")
        n = process()
        val array = IntArray(n)
        i = 0
        while (i < n) {
            print("Nhap phan tu thu " + (i + 1) + " ")
            array[i] = process()
            i++
        }
        println("Cac phan tu trong day xuat hien 2 lan: ")
        i = 0
        while (i < n) {
            if (countElement(array, n, array[i]) == 2 && countElement(array, i, array[i]) == 0) {
                print(" " + array[i])
            }
            i++
        }

    }
}
